pinot-common/src/main/java/org/apache/pinot/common/utils/helix/TableCache.java

```diff
@@ -80,6 +80,14 @@ public String getActualColumnName(String tableName, String columnName) {
     return columnName;
   }
 
+  public TableConfig getTableConfig(String tableName) {
+    return _tableConfigChangeListener._tableConfigMap.get(canonicalize(tableName));
+  }
+
+  private String canonicalize(String name) {
```

Comment:
Using canonicalize explicitly states the reason why we're making the those strings lower case which IMO makes the code more maintainable, but since you both think otherwise, I'm reverting it.